src/field_theory/polynomial_galois_group.lean

```diff
@@ -324,6 +332,86 @@ begin
   rw [aeval_def, map_root_of_splits _ (splitting_field.splits p) hp],
 end
 
+instance {p : polynomial ℚ} : fact (p.splits (algebra_map ℚ ℂ)) :=
```

Comment:
There should be not global `fact` instance in the library, for performance reasons. You can just write this as a lemma, and then add `local attribute [instance] the_lemma_name`.